azure/packages/external-controller/package.json

```diff
@@ -7,7 +7,7 @@
 	"repository": {
 		"type": "git",
 		"url": "https://github.com/microsoft/FluidFramework.git",
-		"directory": "examples/hosts/app-integration/external-controller"
+		"directory": "azure/packages/external-controller"
```

Comment:
could you double-check scripts in each changed package? It looks like the prettier commands use relative paths dependent on package location and some of these moves change the number of folders.

side-note: I'm not sure why we do that or if it's necessary. I'd expect that there should be a way to set prettier up to be more resilient to package moves like this.

Reply:
Good catch - maybe .prettierignore could move under `@fluidframework/build-common`, but for this change I'll just fix up the depths.

Reply:
I see this also bit recently with #15355